tree/ntuple/v7/doc/specifications.md

```diff
@@ -230,7 +230,7 @@ _Size_: If `T` is zero, the number of bytes to read, i.e. the compressed size of
 Otherwise the 16 least-significant bits, i.e bits 0:15, specify the size of the locator itself (see below).
 
 _T(ype)_: Zero for a simple on-disk or in-file locator, 1 otherwise.
-Can be interpreted as the sign bit of the size, i.e. negative sizes indicate non-disk locators.
+Can be interpreted as the sign bit of the size, i.e. negative sizes indicate non-standard locators.
```

Comment:
should probably be consistent and also replace the two other occurrences of `non-disk` below